### PR TITLE
chore: spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/renovatebot-renovate/commit/e1258f6d24a4cff641e8c182ca81dd36f6e1c4d0#commitcomment-43355498

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/renovatebot-renovate/commit/324417cc919c6ceda74a2c36c41ad87076714127

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: lib/manager/bazel/extract.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ function findBalancedParenIndex(longString: string): number {
    * if one opening parenthesis -> 1
    * if two opening parenthesis -> 2
    * if two opening and one closing parenthesis -> 1
-   * if ["""] finded then ignore all [)] until closing ["""] parsed.
+   * if ["""] found then ignore all [)] until closing ["""] parsed.
```
Comment (jsoref) [inline lib/manager/bazel/extract.ts:52]: not sure

File: lib/manager/bazel/update.spec.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ describe('manager/bazel/update', () => {
       expect(res).not.toEqual(content);
     });
 
-    it('updates container_pull deptype and prserves comment', async () => {
+    it('updates container_pull deptype and preserves comment', async () => {
```
Comment (jsoref) [inline lib/manager/bazel/update.spec.ts:59]: not sure

File: lib/manager/npm/extract/common.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import type { PackageJson } from 'type-fest';
 
-export type NpmPackageDependeny = PackageJson.Dependency;
+export type NpmPackageDependent = PackageJson.Dependency;
```
Comment (jsoref) [inline lib/manager/npm/extract/common.ts:3]: Oops, Google Sheets guessed wrong

**Review by jsoref — COMMENTED** [review]
Corrections were picked by Google Sheets -- looks like I got one wrong. fixing shortly

File: docs/usage/config-presets.md [hunk 4]
```diff
@@ -96,7 +96,7 @@ Here is how you would use these in your Renovate config:
 
 ```json
   "extends": [
-    ":labels(depedendencies,devops)",
+    ":labels(dependencies,devops)",
```
Comment (rarkins) [inline docs/usage/config-presets.md:99]: Revert

**Review by rarkins — CHANGES_REQUESTED** [review]
Just one to revert! Thanks

File: docs/usage/config-presets.md (hunk 4 above)
Comment (jsoref) [reply docs/usage/config-presets.md:99]: Sure. Out of curiosity, is this a compatibility thing?

Comment (jsoref) [reply docs/usage/config-presets.md]: (There are only 686 results for this token according to Google vs 169 million for the other. GitHub has 1495 code results vs 72 million.)

**Review by rarkins — APPROVED** [review]

**Pull request merged**